### DISCUSSION TITLE
Running collections/requests using imported data

### DISCUSSION THREAD

**Discussion by DeclanPaulKeane** [Kong/insomnia#7088] [Ideas] [answered]
Allow the ability to import a CSV or JSON file, and use the values from the data file in a collection or series of requests.

**Comment by subnetmarco** [comment]
Coming very soon in the upcoming Insomnia v10.

**Reply by subnetmarco** [reply, marked as answer]
Here is an update and a demo: https://github.com/Kong/insomnia/issues/3294#issuecomment-2322152235
